products/sle/main.pm

```diff
@@ -314,11 +314,17 @@ if (get_var('ALL_MODULES') && sle_version_at_least('15')) {
 }
 
 if (get_var('ENABLE_ALL_SCC_MODULES') && !get_var('SCC_MODULES')) {
-    if (check_var('ARCH', 'aarch64')) {
-        set_var('SCC_ADDONS', 'pcm,tcm');
+    if (sle_version_at_least('15')) {
+        # let's start with what corresponds to server
+        set_var('SCC_ADDONS', 'base,script,desktop,serverapp,phub');
```

Comment:
hm, ok. 'phub' is not really part of what was SLES in before but nevermind as we will change this later anyway